quack_app/test/golden_widget_test.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+import 'package:quack_app/main.dart';
+import 'package:golden_toolkit/golden_toolkit.dart';
+
+void main() {
+
+  testWidgets('Golden test', (WidgetTester tester) async {
```

Comment:
I think this needs to be `testGoldens()` and please make the group name whatever we are testing, i.e the Home Page. Hopefully in the future we can make more specific Goldens Tests, but for now, this general one is fine.

Reply:
Addressed